test/unit/query-generator/json-path-extraction-query.test.ts

```diff
@@ -0,0 +1,49 @@
+import { expectsql, getTestDialect, sequelize } from '../../support';
+
+const dialectName = getTestDialect();
+
+const notSupportedError = new Error(`JSON operations are not supported in ${dialectName}.`);
+
+describe('QueryGenerator#jsonPathExtractionQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  it('should handle isJson parameter true', () => {
+    expectsql(() => queryGenerator.jsonPathExtractionQuery('profile', 'id', true), {
```

Comment:
other test I just thought of that we can // TODO:

if profile starts and ends with the `` ` `` character, or the `"` character (which is perfectly valid, even if an edge case), I'm pretty sure it breaks right now